bin/package-electron

```diff
@@ -34,8 +34,6 @@ if [ "$OSTYPE" == "msys" ]; then
     fi
 fi
 
-yarn rebuild-electron
```

Comment:
We run this before packing now  so we know what the architecture is. Running this here builds it for the build machines architecture.